releases/v0.1.36-dev.yml

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
Would you remind renaming the file name to just `dev.yaml`? Our to be updated release pipeline / job should be responsible for picking a proper name at release time.

Reply:
done @timoreimann, thanks for the review